src/sass/initial/_type.scss

```diff
@@ -92,7 +92,11 @@ strong {
 
   // Container widths here were determined by when the longer color values (p3)
   // would overflow the container
-  @container tool (min-width: 20rem) {
+  @container tool (min-width: 15rem) {
+    --tool-font-size: 5cqi;
+  }
```

Comment:
cqi ok to use?? I only added this new in-between size since it is only between 15 and 21 rem and p3 can be really long.

Reply:
Seems like `cqi` has good support. Nice to see it in use.

Reply:
If we're using container queries, the units should have the same support.